perfkitbenchmarker/network.py

```diff
@@ -102,6 +102,85 @@ def __repr__(self):
     return '%s(%r)' % (self.__class__, self.__dict__)
 
 
+class BaseVPNGW(object):
```

Comment:
Please change VPNGW to VpnGateway where applicable.

Reply:
Fixed.